src/mobility/use-operators.tsx

```diff
@@ -0,0 +1,20 @@
+import {useFirestoreConfiguration} from '@atb/configuration';
+import {FormFactor} from '@atb/api/types/generated/mobility-types_v2';
+
+export const useOperators = () => {
+  const {mobilityOperators} = useFirestoreConfiguration();
+
+  return (types: undefined | FormFactor | FormFactor[]) => {
+    if (!types) {
+      return mobilityOperators ?? [];
+    }
+    types = Array.isArray(types) ? types : [types];
```

Comment:
Mmm, I wonder, If it will be at the end an array why not just (types?: FormFactor[])

Reply:
It's just more convenient from where this function is called. E.g. I can now do like this `useOperators(FormFactor.Scooter)` rather than `useOperators([FormFactor.Scooter])`.